src/Security/Security/help/Az.Security.md

```diff
@@ -1,91 +1,142 @@
 ---
 Module Name: Az.Security
-Module Guid: 5e312bb4-9d3a-4c88-94c3-8e5bbb2e3da4
-Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.security
-Help Version: 0.1.0
+Module Guid: 2b6bac56-a03a-4606-8b05-c1a1f36248db
+Download Help Link: {{ Update Download Link }}
+Help Version: {{ Please enter version of help manually (X.X.X.X) format }}
 Locale: en-US
 ---
 
 # Az.Security Module
 ## Description
-Azure Security Center gives you control over the security of your Azure subscriptions and other machines that you connected to it outside of Azure.
+{{ Fill in the Description }}
```

Comment:
There's something wrong in this file -- contents were replaced by placeholders.
I recommend:
1. Replace this file with the version on master https://github.com/Azure/azure-powershell/blob/master/src/Security/Security/help/Az.Security.md
1. Follow this guide to update markdowns https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module